odpf/optimus/runtime_service.proto

```diff
@@ -221,6 +221,11 @@ service RuntimeService {
             get: "/v1/project/{project_name}/namespace/{namespace}/datastore/{datastore_name}/backup"
         };
     }
+    rpc GetBackupDetail(GetBackupDetailRequest) returns (GetBackupDetailResponse) {
```

Comment:
We can call this just `GetBackup` right?

Reply:
updated to GetBackup